tests/inference/ebm_utils_test.py

```diff
@@ -14,18 +14,21 @@
 # ==============================================================================
 """Tests for qhbmlib.inference.ebm_utils"""
 
+import itertools
```

Comment:
nit: unused?

Reply:
Removed.  Noted the linter is not catching these in #126